docs/01-app/01-getting-started/05-css-and-styling.mdx

```diff
@@ -253,7 +253,7 @@ const nextConfig = {
   },
 }
 
-module.exports = nextConfig
+export default nextConfig
```

Comment:
This would need to be mjs, right?

Reply:
Nice point out!
I checked Node.js [Determining module system](https://nodejs.org/api/packages.html#determining-module-system), it was good to change `js` to `mjs` file extension.

On the other hand, we can still use `js` when the nearest parent `package.json` file contains a top-level ["type"](https://nodejs.org/api/packages.html#type) field with a value of `"module"`.
Anyway, I have updated `mjs`.
Thanks.